R/qenv-constructor.R

```diff
@@ -47,7 +47,7 @@ qenv <- function() {
 #' @aliases new_qenv,environment,missing-method
 #' @aliases new_qenv,missing,missing-method
 #'
-#' @seealso [`get_var()`], [`get_warnings()`], [`join()`], [`concat()`]
+#' @seealso [`base::within()`], [`get_var()`], [`get_warnings()`], [`join()`], [`concat()`]
```

Comment:
The backticks are not necessary for function calls. They generate the same exact documentation

```suggestion
#' @seealso [base::within()], [get_var()], [get_warnings()], [join()], [concat()]
```

Reply:
Neat but I will keep the backticks in case I forget the parentheses at some point.